lib/templates.js

```diff
@@ -289,7 +289,7 @@ const fCreateEventResourceTemplate = (srv, srvDefinition, global,packageIds) =>
  */
 function checkEntityFunctionAction(srvDefinition, global) {
   if (srvDefinition.entities) {
-    return srvDefinition.entities.map((entity) => {
+    return [...srvDefinition.entities].map((entity) => {
```

Comment:
I think there are 5 more places need to be adjust, for example srvDefinition.actions.map. But we don't have corresponding test, lets ignore it in this pr.